app/gui/src/project-view/components/GraphEditor.vue

```diff
@@ -596,15 +598,46 @@ async function handleFileDrop(event: DragEvent) {
   // A vertical gap between created nodes when multiple files were dropped together.
   const MULTIPLE_FILES_GAP = 50
 
+  const payload = ASSET_ROWS.lookup(event)
```

Comment:
@farmaazon here is the implementation of the D&D from the assets table to the editor